test/passes/ignore_missing_func.bin.txt

```diff
@@ -0,0 +1,1067 @@
+(module
```

Comment:
If this is just supposed to test that 0-based ranges are dropped, is it really necessary to dump *all* of the debug info for testing? Why not just use `llvm-dwarfdump -line-tables`?

Reply:
Maybe it's a little unnecessary, but I like seeing that nothing else changes in the output, just to be safe.

If you prefer more compact testcases, we can add options for what flags to pass into dwarfdump.

Reply:
Generally it's nice to have more targeted tests, so that when you make an unrelated change, it doesn't require updating a ton of unrelated stuff. but now that you mention it, not very many of Binaryen's tests are targeted. so this probably just fits with the rest?
Anyway we'll probably want to update all the generated dwarf tests soon anyway, so we can defer that.